packages/web/test/mocks/tx.ts

```diff
@@ -1,4 +1,10 @@
-import algosdk from "algosdk";
+import algosdk, { LogicSigAccount } from "algosdk";
+import { testnetURL } from "../../src";
+import { algoexplorerAlgod } from "../../src/lib/api";
+import { HttpNetworkConfig } from "../../src/types";
```

Comment:
show we import it from @algob/web?

Reply:
but in all the other tests we i,port like this only )like algob or runtime) because we don't have the dependency of algob/web in package.json of web package

Reply:
I think both options are good enough.